scripts/includes/rhel.sh

```diff
@@ -9,35 +9,42 @@ install_yum_utils() {
 get_full_pkg_versions() {
   if [ "$VERSION" != '' ];
   then
-    local ST2_VER=$(repoquery -y --nvr --show-duplicates st2 | grep -F st2-${VERSION} | sort --version-sort | tail -n 1)
+    local RHMAJVER=`cat /etc/redhat-release | sed 's/[^0-9.]*\([0-9.]\).*/\1/'`
+    local REPOQUERY_FLAGS="--nvr --show-duplicates"
+    if [ "$RHMAJVER" -ge "8" ]; then
+      # RHEL 8 and newer, you need "-y" flag to avoid being prompted to confirm "yes"
+      local REPOQUERY_FLAGS="-y ${REPOQUERY_FLAGS}"
```

Comment:
So what if instead of bundling all the args together... you did something like:
if el8 then:
`DASH_YES="-y"`
else
`DASH_YES=""
fi

This would retain the flags in place and help the next reader better understand what's going on, and be a less complex change.

Reply:
I agree with approach that `VAR="-y"` would look more readable and straightforward :+1: 

---

Personally I would remove this part from templating alltogether to have a dedicated block for each platform as it became too spammy. It's not normal when EL6 and EL7 bash file include EL8 checks.